testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/JBossNodeNameTestCase.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.test.manualmode.management;
+
+import static org.jboss.as.test.manualmode.expressions.module.TestSecureExpressionsExtension.PARAM_EXPRESSION;
```

Comment:
I would say it is better to keep tests independent; `TestSecureExpressionsExtension` is unrelated to what we are doing here, so I would not use a constant defined in an external class, instead it would be better to define it in this class.

Do you agree?

Reply:
fixed